resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/ResourceValidator.scala

```diff
@@ -18,28 +18,33 @@ import org.broadinstitute.dsde.workbench.openTelemetry.OpenTelemetryMetrics
 object ResourceValidator {
   def run[F[_]: ConcurrentEffect: Parallel](isDryRun: Boolean,
                                             shouldRunAll: Boolean,
-                                            shouldRunCheckDeletedRuntimes: Boolean,
-                                            shouldRunCheckErroredRuntimes: Boolean,
-                                            shouldRunCheckDeletedDisks: Boolean,
-                                            shouldRunCheckInitBuckets: Boolean,
+                                            shouldCheckDeletedRuntimes: Boolean,
```

Comment:
let's not re-order these params around much...they're all `Boolean`, which means if you get the order wrong, we can't know at compile time

Reply:
Understood. Part of my motivation was to reduce future errors too, by grouping them more logically.